apps/docs/src/guide/transactions/index.md

```diff
@@ -0,0 +1,13 @@
+# Transactions
+
+A transaction is any interaction with the blockchain. This is the primary way of interacting with the Fuel Network and can include actions like transferring assets, deploying contracts and minting tokens.
```

Comment:
Not all interactions with the blockchain are transactions, e.g. `provider.operations.getChain()`. Nothing comes to mind for rephrasing it, though.

Reply:
Hmmm interesting, I'd originally taken that from [here](https://docs.fuel.network/docs/intro/glossary/#transaction).

"A transaction is a way of interacting with the blockchain"?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1961/commits/5fcb542f03019c177c4234ea07a05c343dddd034